echo/echo_integration_test.go

```diff
@@ -44,7 +44,6 @@ func TestStatusCodes(t *testing.T) {
 	const configFile = `verbosity: debug
 strictmode: false
 network:
-  disablenodeauthentication: true
```

Comment:
I thought this change was reverted?

Reply:
I reinstated the dummy authenticator for disableTLS. Removal of disablenodeauthentication is still the goal of this PR.